src/integrationTest/java/uk/gov/hmcts/darts/ws/ContextRegistryParent.java

```diff
@@ -0,0 +1,324 @@
+package uk.gov.hmcts.darts.ws;
+
+import documentum.contextreg.LookupResponse;
+import documentum.contextreg.RegisterResponse;
+import documentum.contextreg.UnregisterResponse;
+import lombok.Getter;
+import org.junit.jupiter.api.Assertions;
+import uk.gov.hmcts.darts.cache.token.config.CacheProperties;
+import uk.gov.hmcts.darts.cache.token.exception.CacheException;
+import uk.gov.hmcts.darts.utils.CacheUtil;
+import uk.gov.hmcts.darts.utils.IntegrationBase;
+import uk.gov.hmcts.darts.utils.TestUtils;
+import uk.gov.hmcts.darts.utils.client.SoapAssertionUtil;
+import uk.gov.hmcts.darts.utils.client.ctxt.ContextRegistryClient;
+
+import java.net.URI;
+import java.net.URL;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+public class ContextRegistryParent extends IntegrationBase {
+
+    void executeTestGetContextRegistryWsdl() throws Exception {
+        HttpRequest request = HttpRequest.newBuilder()
+                .uri(new URI(getGatewayUri() + "runtime/ContextRegistryService?wsdl"))
+                .timeout(Duration.ofMinutes(2))
+                .header("Content-Type", "application/json")
```

Comment:
Is this the right header value?

Reply:
Nope. Superflous for the GET anyhow. Removing. 

Done